pages/projects/submit.tsx

```diff
@@ -45,7 +45,8 @@ const SubmitProject = () => {
   if (status === 'loading') {
     return (
       <div className="h-screen flex items-center justify-center">
-        Loading...
+        {/* Loading... */}
+        <iframe src="https://embed.lottiefiles.com/animation/98194"></iframe>
```

Comment:
could you use lottie package which exists in our current project to display the animation?
we have implemented for the homepage, see it for reference.

Reply:
> could you use lottie package which exists in our current project to display the animation? we have implemented for the homepage, see it for reference.

The triangular loader which is present at right bottom of page?